Dockerfile

```diff
@@ -9,24 +10,18 @@ WORKDIR /code
 
 # Create conda environment
 COPY environment.yml .
-RUN mamba env create -n raven -f environment.yml && mamba install -n raven gunicorn  && mamba clean --all --yes
+RUN mamba env create -n raven -f environment.yml && mamba install -n raven gunicorn && mamba clean --all --yes
 
 # Add the raven conda environment to the path
 ENV PATH=/opt/conda/envs/raven/bin:$PATH
 
-# Copy raven source code
+# Copy WPS project
 COPY . /code
 
-# Install raven
+# Install WPS project
 RUN pip install . --no-deps
 
 # Start WPS service on port 9099 on 0.0.0.0
 EXPOSE 9099
 
-CMD ["gunicorn", "--bind=0.0.0.0:9099", "raven.wsgi:application"]
-#CMD ["exec raven-wps start -b '0.0.0.0' -c etc/demo.cfg"]
-
-# docker build -t pavics/raven .
-# docker run -p 9099:9099 pavics/raven
-# http://localhost:9099/wps?request=GetCapabilities&service=WPS
-# http://localhost:9099/wps?request=DescribeProcess&service=WPS&identifier=all&version=1.0.0
```

Comment:
Why remove those manual step comment?  I think it's a reminder for those who wants to test locally on his machine, without having to push bogus commits just to trigger Github pipeline.